conf/config-default.yaml

```diff
@@ -128,6 +128,7 @@ nginx_config:                     # config for render the template to genarate n
   #envs:                            # allow to get a list of environment variables
   #  - TEST_ENV
   http:
+    enable_access_log: true
```

Comment:
For the compatibilities' sake, the access_log should be enabled even if this field absent.

Reply:
agree

Reply:
OK, I will update this PR later.